dds_cli/utils.py

```diff
@@ -53,13 +53,19 @@ def calculate_magnitude(projects, keys, iec_standard=False):
             # exclude values smaller than base, such that empty projects don't interfer with
             # the calculation ensures that a minimum can be calculated if no val is larger than base
             minimum = (lambda x: min(x) if x else 1)([val for val in values if val >= base])
-            mag = 0
+            maximum = (lambda x: max(x) if x else 1)([val for val in values if val >= base])
+            magmin = magmax = 0
```

Comment:
General point - I'm always scared of doing this because of Python's mutability, the same thing that @talavis was just talking about in Slack about function defaults.

It's fine here because we're setting to an int which is immutable, however if you did `x = y = {}` then `x` would change if you changed `y`, because dicts are mutable.

https://note.nkmk.me/en/python-multi-variables-values/

> Be careful when assigning mutable objects such as `list` or `dict` instead of immutable objects such as `int`, `float`, or `str`.

Reply:
Thanks for the explanation. I will change that. 

It is the first time that I wrote something like this. Given Pythons amazing brevity, I believed that would be the "Pythonic" way to assign one value to two variables.

I thought, I had seen such a notation somewhere in our code before, but I must have mixed it up with this: 

https://github.com/ScilifelabDataCentre/dds_web/blob/4549858baef0eac95b0d7d0d9a3803e074af4e22/dds_web/api/user.py#L287

Reply:
Yes exactly, it is quite nice any Pythonic. Just given our recent conversation about mutability causing problems I thought it was a good point to bring up for future coding :)